scripts/perfmon.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Performance monitoring/alarm generation daemon
+After=xcp-rrdd.service xapi.service
+Wants=xcp-rrdd.service xapi.service
```

Comment:
I think that these `After`s and `Wants`s are not strictly needed. Perfmon will most likely just retry.